source/features/more-dropdown.tsx

```diff
@@ -24,7 +24,7 @@ export function onlyShowInDropdown(id: string): void {
 		return;
 	}
 
-	(tabItem!.closest('li') ?? tabItem!.closest('.UnderlineNav-item'))!.remove();
+	(tabItem!.closest('li') ?? tabItem!.closest('.UnderlineNav-item'))!.classList.add('d-none');
```

Comment:
Can also be done with `.hidden = true`, which I generally prefer since it's native, but it's _ok_ either way.

Reply:
Will require `as HtmlElelemnt` worth it?

Reply:
Nah, but still need https://github.com/sindresorhus/refined-github/issues/4008#issuecomment-823814526